ikarus/finiteelements/mechanics/nonlinearelastic.hh

```diff
@@ -230,29 +232,34 @@ private:
       return mat_;
   }
 
-protected:
+  bool usesEASSkill() const {
+    if constexpr (hasEAS)
```

Comment:
We can also disable all the linear elastic functionality if eas is present and the eas method always deals with everything. Like `calculateMatrix(...) requires not hasEAS` which would disable the function alltogether and we would not have the runtime switch with `usesEASSkill()`.